packages/language-service/package.json

```diff
@@ -28,7 +28,7 @@
     "unified"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "tsc --build --clean && tsc --build",
```

Comment:
why the double build?

Reply:
`--build --clean` removes the `.d.ts` files it would overwrite when running `--build`. This basically means it compiles without any outdated type definitions causing errors.